README.md

```diff
@@ -1 +1,64 @@
-# Bucketeer CLI
+# Bucketeer CLI &nbsp;[![Build Status](https://travis-ci.com/UCLALibrary/bucketeer-cli.svg?branch=main)](https://travis-ci.com/UCLALibrary/bucketeer-cli)
+
+Uploads CSV files to the [Bucketeer](https://github.com/UCLALibrary/bucketeer) service for processing.
+
+## Installation
+
+First, ensure that you have Bash, cURL, Python 3.6+ and Pip installed on your system.
+
+When that's done, clone the repository and follow the development instructions below (install script coming soon).
```

Comment:
Are you going to create a new ticket for this?

Reply:
IIIF-939